docs/totest.asciidoc

```diff
@@ -26,27 +26,85 @@ of the project openSUSE:Factory:ToTest, if the version is different
 from the current Factory.
 
 
-Installation
+Setup
 ------------
-[id="install"]
+[id="cli"]
 
-Clone this repository:
+Create a :ToTest subproject of the project you like to have ttm running. The
+user account of the ttm needs to have access, eg.
 
 -------------------------------------------------------------------------------
-git clone https://github.com/openSUSE/openSUSE-release-tools.git
+$ osc meta prj MyProject:ToTest
+[...]
+<person userid="totest-manager" role="maintainer"/>
+[...]
 -------------------------------------------------------------------------------
 
-Link the plugin and the library in +~/.osc-plugins+ or +/var/lib/osc-plugins+.
+If images or products are meant to be released make sure to set the image type:
 
 -------------------------------------------------------------------------------
-cd openSUSE-release-tools
-ln -sr ./osc-totest.py ./osclib ~/.osc-plugins
+$ osc meta prjconf MyProject:ToTest
+%if "%_repository" == "images"
+%Type: kiwi
+%Repotype: staticlinks
+%Patterntype: none
+%Prefer: openSUSE-release
+%endif
 -------------------------------------------------------------------------------
 
+In the main project meta, add release targets for the repos that you want to
+have released into :ToTest
 
-Command line
-------------
-[id="cli"]
+-------------------------------------------------------------------------------
+$ osc meta prj MyProject
+[...]
+<repository name="images">
+  <releasetarget project="MyProject:ToTest" repository="images" trigger="manual"/>
+[...]
+-------------------------------------------------------------------------------
+
+Configure the attribute for ttm so it knows what to release:
+
+-------------------------------------------------------------------------------
+$ cat <<EOF > myattribute
+jobs_num: 2
+openqa_group: My Product 66.6
+openqa_server: https://openqa.opensuse.org
+product_repo: images
+product_arch: x86_64
+products:
+  ftp:
+  - 000product:MyProduct-ftp-ftp-x86_64
+  main:
+  - 000product:MyProduct-dvd5-dvd-x86_64
+  images:
+  - MyProduct-Live:
+      - x86_64
+take_source_from_product: true
+need_same_build_number: true
+set_snapshot_number: true
+test_subproject: ToTest
+EOF
+$ osc meta attribute MyProject -c -a OSRT:ToTestManagerConfig -F myattribute
+-------------------------------------------------------------------------------
+
+Create a 00Meta package in the main project for ttm to store some metadata:
```

Comment:
I don't think ttm stores anything in 00Meta

Reply:
it did for me, otherwise I would not have created 00Meta:

`ttm/manager.py:        self.api.pseudometa_file_ensure(self.version_file(target), version,
ttm/manager.py:        return self.api.pseudometa_file_load(self.version_file('totest'))
ttm/publisher.py:            self.update_status('publishing', self.api.pseudometa_file_load(self.version_file('snapshot')))`

Reply:
I see - that's `openSUSE:%(project)s:Staging/dashboard` otherwise

Reply:
It's the `pseudometa_package` setting in the config which is set to `:Staging/dashboard` for the main projects but defaults to `00Meta` https://github.com/openSUSE/openSUSE-release-tools/blob/027727ccf3d1d0d4dc16ce3f3e4954cb725e17a5/osclib/core.py#L322